config/nginx.conf

```diff
@@ -0,0 +1,50 @@
+server {
+    listen       80;
+    server_name  localhost;
+
+    #access_log  /var/log/nginx/host.access.log  main;
+
+    location / {
+        root   /usr/share/nginx/html;
+        index  index.html index.htm;
+    }
+
+    location /nginx_status {
+        stub_status;
+        allow all;
+        # allow 127.0.0.1;
+        # deny all;
+    }
+
+    #error_page  404              /404.html;
+
+    # redirect server error pages to the static page /50x.html
+    #
+    error_page   500 502 503 504  /50x.html;
+    location = /50x.html {
+        root   /usr/share/nginx/html;
+    }
+
+    # proxy the PHP scripts to Apache listening on 127.0.0.1:80
```

Comment:
This could probably be cleaned up.

Reply:
I thought about that as well... 
I created this file using 
```
docker run --rm -it docker.io/library/nginx:1.23.3-alpine cat /etc/nginx/conf.d/default.conf
```
so was on the fence leaving it like this or clean-up... thanks for the input